src/data/nat/log.lean

```diff
@@ -64,4 +93,19 @@ end
 lemma pow_log_le_self (b x : ℕ) (hb : 1 < b) (hx : 1 ≤ x) : b ^ log b x ≤ x :=
 by rw [pow_le_iff_le_log _ _ hb hx]
 
+lemma log_le_log_of_le {b n m : ℕ} (h : n ≤ m) : log b n ≤ log b m :=
+begin
+  by_cases hb : b ≤ 1, by rw log_eq_zero_of_lt' hb; exact zero_le',
+  by_cases hn : n = 0, by rw [hn, log_zero_eq_zero]; exact zero_le',
+
+  rw ←pow_le_iff_le_log _ _ (not_le.mp hb) (lt_of_lt_of_le (zero_lt_iff.mpr hn) h),
+  exact (pow_log_le_self b n (not_le.mp hb) (zero_lt_iff.mpr hn)).trans h,
+end
+
+lemma log_le_log_succ {b n : ℕ} : log b n ≤ log b n.succ :=
+log_le_log_of_le $ le_succ n
+
+lemma log_mono {b : ℕ} : monotone (λ n : ℕ, log b n) :=
+monotone_of_monotone_nat $ λ n, log_le_log_succ
```

Comment:
Can you prove the stronger statement with [`strict_mono`](https://leanprover-community.github.io/mathlib_docs/order/basic.html#strict_mono) instead?